Calculator/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using CalculatorLibrary;
+using Newtonsoft.Json;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool endApp = false;
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
```

Comment:
This comment doesn't provide much information. These lines could instead be separated into their own method with a name something like `DisplayTitle`. 
That way it would be called by doing `DisplayTitle()` and it would be self explanatory.